management_app/templates/faculty/members.html

```diff
@@ -0,0 +1,135 @@
+{% extends 'base.html' %}
+
+{% block title %}Faculty{% endblock %}
+{% block header %}Faculty Members{% endblock %}
+
+{% block header_content %}
+  <nav class="navbar navbar-expand-sm bg-light">
+    <div class="container-fluid">
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="/faculty/points">Points</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="/faculty/members">Members</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+{% endblock %}
+
+{% block content %}
+<div class="d-grid justify-content-end">
+  <div class="d-grid justify-content-end">
+    <div class="d-inline-flex">
+      <button style="margin-right: 20px;" type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#importChange">
+        <i class="fa-solid fa-download"></i>
+        Import Changes
+      </button>
+      <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#addModal">
+        <a class="dropdown-item" href="{{ url_for('faculty.create_member') }}"><i class="fa-solid fa-plus"></i> Add + Faculty Member</a>
```

Comment:
Change `Add + Faculty Member` to `Add Faculty Member`.

Reply:
Updated in the new commit.